study.md

```diff
@@ -129,7 +140,8 @@ Write some JavaScript code that uses the array `filter()` method to duplicate
 this functionality.
 
 ```javascript
-// your answer here
+let numbers = [1,2,3,4,5,6,7,8,9,10];
+let filteredNumbers = filter((value)=>x%3===0)
```

Comment:
`filter` needs to be invoked on `numbers`.  The anonymous function also names `value` as its parameter but the return value uses `x`, which is undefined.